packages/expo-web-browser/expo-module.config.json

```diff
@@ -2,5 +2,8 @@
   "platforms": ["ios", "android"],
   "ios": {
     "modulesClassNames": ["WebBrowserModule"]
+  },
+  "android": {
+    "modulesClassNames": ["expo.modules.webbrowser.WebBrowserModule"]
```

Comment:
Fyi: we've renamed this field to just `modules`, but we didn't change existing configs for backwards compatibility. No need to change it here right now, we'll do this once for each package at some point before next release.